framework/common/test/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ if(BUILD_TESTS AND NOT MSGLIB STREQUAL "None")
 		install(TARGETS ${target} DESTINATION bin/tests/common)
 		add_test(NAME ${target} COMMAND $<TARGET_FILE:${target}>)
 	endforeach(test ${tests})
-else()
+elseif(BUILD_TESTS)
```

Comment:
Nitpick: it would be better to have an outer condition BUILD_TESTS and then an inner condition on MSGLIB, which allows to refactor a bit because the code below will always be there so outside of the MSGLIB condition.

Reply:
It's ugly either way, but OK I guess, I'll refactor it :)